<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchByAuthor</class>
 <widget class="QDialog" name="SearchByAuthor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-radius: 5px;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 5px;</string>
   </property>
   <property name="title">
    <string>SEARCH LIBRARY ITEM BY AUTHOR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>601</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color:rgb(255, 2, 61);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Go back to home</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
